doc/contributing/pull-requests.md

```diff
@@ -184,6 +184,10 @@ A good commit message should describe what changed and why.
    of the log. Use the `Fixes:` prefix and the full issue URL. For other
    references use `Refs:`.
 
+   `Fixes:` references get automatically added to your commit when the PR
+   is merged, so it is sufficient to just link your PR with the issue or issues it is
```

Comment:
The rest should be adjusted but fixing it through Github UI is extremely hard for me at the moment.

```suggestion
   is merged. It is sufficient to add a link your PR description.
```

Reply:
Well on GitHub you can also link the issue without adding it to your PR description 🙂 
(although I don't know if someone really prefers that option 😅)

Reply:
@anonrig I had to basically revert your suggestion because it was breaking the sentence:
![Screenshot 2025-01-24 at 01 50 53](https://github.com/user-attachments/assets/11352728-2fb2-4190-9226-5e4bb1b1b0d3)

I've anyways slightly tweaked the wording, please have a look and let me know if it is any better 🙂

Reply:
This has been updated as part of the suggested changes from @aduh95 👍